Weather App/ViewModel/Utility.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Utility.swift
+//  Weather App
+//
+//  Created by Shikhar Sharma on 22/03/22.
+//
+
+import Foundation
+
+extension String {
+    func getImageLocationFromUrl () -> String{
+        let ch = self[self.index(self.endIndex, offsetBy: -11)]
+        if ch == "d" {
+            return String(self[self.index(self.endIndex, offsetBy: -11)..<self.index(self.endIndex, offsetBy: -4)])
+        }
+        return String(self[self.index(self.endIndex, offsetBy: -13)..<self.index(self.endIndex, offsetBy: -4)])
+    }
+    func lastFiveSubstring () -> String {
+        return String(self[self.index(self.endIndex, offsetBy: -5)..<self.endIndex])
```

Comment:
This will crash for strings with less than 5 characters

Reply:
added guard